music21/harmony.py

```diff
@@ -1458,8 +1504,8 @@ class ChordSymbol(Harmony):
     (<music21.pitch.Pitch A-2>, <music21.pitch.Pitch C-3>, <music21.pitch.Pitch E-3>)
 
     >>> harmony.ChordSymbol('F-dim7').pitches
-    (<music21.pitch.Pitch F-2>, <music21.pitch.Pitch A--2>,
-     <music21.pitch.Pitch C--3>, <music21.pitch.Pitch E---3>)
+    (<music21.pitch.Pitch F-2>, <music21.pitch.Pitch A--2>, \
+<music21.pitch.Pitch C--3>, <music21.pitch.Pitch E---3>)
```

Comment:
Why this change?  the documentation will look worse with it.

Reply:
I'm not sure why I need this, it's just that the doctest fails otherwise ([as described here](https://stackoverflow.com/questions/13395610/wrapping-python-doctest-results-that-are-longer-than-80-characters) for example). Maybe it's just something with my local config, I'll remove it and see if it still passes.

Reply:
Music21 doctests (when run through its system) are space agnostic -- it doesn't matter if there are extra spaces or not.